dot.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="dot.css">
+    <script src="dot.js"> </script>
+  </head>
+  <body>
+    <div class="appbar">
+      <div class="left"></div>
+      <div class="middle"></div>
+      <div class="right">
+        <div id="myMenu" class="menu1">
+          <div class="button_div">
+            <button onclick="myFunc()" id="myButton" class="button1"></button>
```

Comment:
Defining a function here makes it a global function, which is very bad practice. Use `addEventListener` instead.